### PR TITLE
Bugfix667

### PR DESCRIPTION
**Pull request by AlexeYankov**
fix adaptive design profile, settings (except payments table)

### CODE REVIEW

File: src/components/Modals/SubscriptionsModal/SubscriptionsModal.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const SubscriptionsModal: React.FC<Props> = ({
                   height={36}
                   className={s.modal__content__avatar}
                 />
-                <p>URLProfiele</p>
+                <p>URLProfielgwegwergrgety45h454545yh45y</p>
```
Comment (LovaVikasso) [inline src/components/Modals/SubscriptionsModal/SubscriptionsModal.tsx:52]: is it ok? URLProfielgwegwergrgety45h454545yh45y

File: src/components/Header/HeaderMenuMobile/HeaderMenuMobile.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  const tf = useTranslations('Header');
+  const router = useRouter();
+  const onBlurHandler = (e: React.FocusEvent<HTMLTextAreaElement, Element>) => {
+    if (e.relatedTarget?.id === 'mobileMenu') {
+      return;
+    }
+    setModal(false);
+  };
+
+  const keyDownHandler = (e: React.KeyboardEvent<HTMLTextAreaElement>) => {
```
Comment (LovaVikasso) [inline src/components/Header/HeaderMenuMobile/HeaderMenuMobile.tsx:35]: you can import KeyboardEvent, useState and MouseEvent without React

**Review by LovaVikasso — APPROVED** [review]

File: src/components/Header/HeaderMenuMobile/HeaderMenuMobile.module.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
+@use '@/app/mixins.scss' as s;
```
[the remark this answers is not in the document]
Comment (AlexeYankov) [reply src/components/Header/HeaderMenuMobile/HeaderMenuMobile.module.scss:1]: line 62))

File: src/components/Header/HeaderMenuMobile/HeaderMenuMobile.tsx (hunk 2 above)
Comment (AlexeYankov) [reply src/components/Header/HeaderMenuMobile/HeaderMenuMobile.tsx:35]: thanks, fix it in next task (refactor)

File: src/components/Header/HeaderMenuMobile/HeaderMenuMobile.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                <span>{tf('mobileMenu.' + ref)}</span>
+              </li>
+            );
+          })}
+          <textarea
+            autoFocus
+            onKeyDown={keyDownHandler}
+            name="post"
+            onBlur={onBlurHandler}
+            style={{
```
[the remark this answers is not in the document]
Comment (AlexeYankov) [reply src/components/Header/HeaderMenuMobile/HeaderMenuMobile.tsx:97]: easy custom modal)

File: src/components/Modals/SubscriptionsModal/SubscriptionsModal.tsx (hunk 1 above)
Comment (AlexeYankov) [reply src/components/Modals/SubscriptionsModal/SubscriptionsModal.tsx:52]: its for test) will fix it

**Pull request merged**